custom_components/tahoma/climate_devices/atlantic_heat_recovery_ventilation.py

```diff
@@ -0,0 +1,247 @@
+"""Support for AtlanticHeatRecoveryVentilation."""
+import logging
+from typing import List, Optional
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    FAN_AUTO,
+    HVAC_MODE_FAN_ONLY,
+    SUPPORT_FAN_MODE,
+    SUPPORT_PRESET_MODE,
+)
+from homeassistant.const import EVENT_HOMEASSISTANT_START, STATE_UNKNOWN, TEMP_CELSIUS
+from homeassistant.core import callback
+from homeassistant.helpers.event import async_track_state_change
+
+from ..coordinator import TahomaDataUpdateCoordinator
+from ..tahoma_entity import TahomaEntity
```

Comment:
I replace TahomaDataUpdateCoordinator and TahomaEntity with OverkizDataUpdateCoordinator and OverkizEntity.

Now, my heaters are back to normal and the VMC appears as device. But it appears as unavailable.
![image](https://user-images.githubusercontent.com/11087736/153655828-2742bfac-47c2-4f1f-8451-e4e37cafe34c.png)

Reply:
There is a change with this command too : 
        await self.async_execute_command(
        await self.executor.async_execute_command(

I added "executor" on each command and now my VMC is back online!
![image](https://user-images.githubusercontent.com/11087736/153659598-95977fb9-5071-4d1e-81b9-89f82c88cdc6.png)

Reply:
Hello @tillstaff, I want to try this branch to test the AtlanticHeatRecoveryVentilation implementation with my VMC (Optimocozy HR Plus).

I simply tried to download the branch and add it in custom_components folder, but still no 'controls' over my ACHRV_Actuator, just sensors. (With and w/o the replacement of `TahomaDataUpdateCoordinator` and `TahomaEntity` with `OverkizDataUpdateCoordinator` and `OverkizEntity` + `const.py` modification).

Is the current state of this branch OK to test?
Thank you